test-dts/ref.test-d.ts

```diff
@@ -0,0 +1,93 @@
+import { Ref, ref, isRef, unref, reactive, expectType } from './index';
+
+function plainType(arg: number | Ref<number>) {
+  // ref coercing
+  const coerced = ref(arg);
+  expectType<Ref<number>>(coerced);
+
+  // isRef as type guard
+  if (isRef(arg)) {
+    expectType<Ref<number>>(arg);
+  }
+
+  // ref unwrapping
+  expectType<number>(unref(arg));
+
+  // ref inner type should be unwrapped
+  const nestedRef = ref({
+    foo: ref(1),
+  });
+  expectType<Ref<{ foo: number }>>(nestedRef);
+  expectType<{ foo: number }>(nestedRef.value);
+
+  // ref boolean
+  const falseRef = ref(false);
+  expectType<Ref<boolean>>(falseRef);
+  expectType<boolean>(falseRef.value);
+
+  // ref true
+  const trueRef = ref<true>(true);
+  expectType<Ref<true>>(trueRef);
+  expectType<true>(trueRef.value);
+
+  // tuple
+  // Not supported by Vue 2
+  // expectType<[number, string]>(unref(ref([1, '1'])));
```

Comment:
This should be supported 🤔

Reply:
Oh. You are right, I forget to pull the head. Thanks